src/lib/pipeline/m2/animation-manager.js

```diff
@@ -1,9 +1,14 @@
+import EventEmitter from 'events';
 import THREE from 'three';
 
-class AnimationManager {
+class AnimationManager extends EventEmitter {
 
   constructor(root, animationDefs, sequenceDefs) {
-    this.animationDefs = animationDefs;
+    super();
+
+    // Complicated M2s may have far more than 10 (default listener cap) M2Materials subscribed to
+    // the same texture animations.
+    this.setMaxListeners(150);
```

Comment:
I don't know what the exact value should be, but 10 is insufficient, even with sharing. There are something like 30 materials used on the burning ship M2 in Howling Fjord.

I'd think there's no real harm in jacking the number up, as you get a warning if you start to exceed it.